README.md

```diff
@@ -46,155 +45,104 @@ The Pravega Operator manages Pravega clusters deployed to Kubernetes and automat
 
 NAME                                            TYPE        CLUSTER-IP    EXTERNAL-IP   PORT(S)              AGE
-service/bar-bookie-headless                 ClusterIP   None          <none>        3181/TCP             2m
-service/bar-pravega-controller              ClusterIP   10.23.244.3   <none>        10080/TCP,9090/TCP   2m
-service/bar-pravega-segmentstore-headless   ClusterIP   None          <none>        12345/TCP            2m
-
-NAME                                                    DESIRED   CURRENT   READY   AGE
-replicaset.apps/bar-pravega-controller-64ff87fc49   1         1         1       2m
+service/pravega-bookie-headless                 ClusterIP   None          <none>        3181/TCP             2m
 
 NAME                                            DESIRED   CURRENT   AGE
-statefulset.apps/bar-bookie                 3         3         2m
-statefulset.apps/bar-pravega-segmentstore   3         3         2m
+statefulset.apps/pravega-bookie                 3         3         2m
 ```
 
-By default, a `PravegaCluster` instance is only accessible within the cluster through the Controller `ClusterIP` service. From within the Kubernetes cluster, a client can connect to Pravega at:
-
-```
-tcp://<pravega-name>-pravega-controller.<namespace>:9090
-```
-
-And the `REST` management interface is available at:
-
+By default, a `BookkeeperCluster` is reachable using this kind of headless service URL for each pod:
 ```
-http://<pravega-name>-pravega-controller.<namespace>:10080/
+http://pravega-bookie-0.pravega-bookie-headless.pravega-bookie:3181
 ```
 
-Check out the [external access documentation](doc/external-access.md) if your clients need to connect to Pravega from outside Kubernetes.
+### Scale a Bookkeeper cluster
 
-### Scale a Pravega cluster
+You can scale Bookkeeper cluster by updating the `replicas` field in the BookkeeperCluster Spec.
 
-You can scale Pravega components independently by modifying their corresponding field in the Pravega resource spec. You can either `kubectl edit` the cluster or `kubectl patch` it. If you edit it, update the number of replicas for BookKeeper, Controller, and/or Segment Store and save the updated spec.
-
-Example of patching the Pravega resource to scale the Segment Store instances to 4.
+Example of patching the Bookkeeper Cluster resource to scale the SegmentStore instances to 4.
 
 ```
-kubectl patch PravegaCluster <pravega-name> --type='json' -p='[{"op": "replace", "path": "/spec/pravega/segmentStoreReplicas", "value": 4}]'
+kubectl patch bk pravega-bookkeeper --type='json' -p='[{"op": "replace", "path": "/spec/replicas", "value": 4}]'
 ```
 
-### Upgrade a Pravega cluster
+### Upgrade a Bookkeeper cluster
 
 Check out the [upgrade guide](doc/upgrade-cluster.md).
 
-### Uninstall the Pravega cluster
+### Uninstall the Bookkeeper cluster
 
 ```
-$ helm delete bar --purge
-$ kubectl delete -f ./example/pvc-tier2.yaml
+$ helm delete pravega --purge
```

Comment:
`pravega` -> `bookkeeper`?

Reply:
bookeeper cluster name is "pravega", so thats correct. But yes from a readability perspective I could change it to "pravega-bk"

Reply:
Fixed